### PR TITLE
feat(editor): block UI in NDV when workflow is listening to events

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

**Review by MiloradFilipovic — DISMISSED** [review]
Looks good. I left some comments in the [linear ticket](https://linear.app/n8n/issue/N8N-4598#comment-8a03930b) that need to be addressed before merging.

File: packages/design-system/src/components/N8nBlockUi/BlockUi.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<template>
+	<transition name="fade" mode="out-in">
+		<div v-show="show" :class="['n8n-block-ui', $style.uiBlocker]" role="dialog" :aria-hidden="true" />
+	</transition>
+</template>
+
+<script lang="ts">
```
Comment (mutdmour) [inline packages/design-system/src/components/N8nBlockUi/BlockUi.vue:7]: nitpick - but we should move towards composition api for new components.. no need to update this for now.. just something to keep in mind

**Review by mutdmour — DISMISSED** [review]
LGTM

File: packages/design-system/src/components/N8nBlockUi/BlockUi.vue (hunk 1 above)
Comment (cstuncsik) [reply packages/design-system/src/components/N8nBlockUi/BlockUi.vue]: done, this component has only 1 prop so it even got less lines of code in no time (refactoring took 1 minute)

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.201.0`

**Pull request merged**